content/home/usage-left.md

```diff
@@ -1,4 +1,4 @@
 ![Developers](/images/home/icon-tags.svg)
 
 **Developers** \
-to focus on code instead of writing tons of configuration code
+to focus on code instead of writing tons of configuration cod
```

Comment:
"code" is correct, "cod" is wrong. So there was no typo to fix.

Reply:
After looking up "cod" and thinking about what we are trying to say here, I would suggest we remove the last word and just say

"to focus on code instead of writing tons of configurations"

Reply:
@christian-kreuzberger-dtx @dirkwall 

Should we use `configurations` or `configuration`. I see that in https://github.com/keptn/keptn.github.io/pull/383, this was corrected to the latter.